haystack/nodes/file_converter/markdown.py

```diff
@@ -27,6 +28,7 @@ def __init__(
         progress_bar: bool = True,
         remove_code_snippets: bool = True,
         extract_headlines: bool = False,
+        add_frontmatter_to_meta: bool = False,
```

Comment:
Please document this parameter in the docstrings.

Reply:
lol, great minds thing alike. It's in the next commit :D